### PR TITLE
Added functionality to urlencode the Mongo username and password

### PR DESCRIPTION
**Pull request by manish535**
…nstall.sh file

### CODE REVIEW

File: deploy/install.sh [hunk 1]
```diff
@@ -245,7 +264,14 @@ echo "Generating the configuration files from the templates"
 . ./template/docker-compose.yml.sh
 . ./template/mongo-init.js.sh
 . ./template/init-letsencrypt.sh.sh
-. ./template/docker.env.sh
+
+if [ 1 ];then
```
Comment (mohanarpit) [inline deploy/install.sh:268]: @manish535 Why not just have 2 variables? One for the url encoded version & 1 for the plain text version. Then in the `docker.env` template, we bind the normal version while in `docker-compose.env` template, we bind the encoded version. 

There is no point of adding an `if` condition for something that will always evaluate to `true`

File: deploy/aws/boot.sh [hunk 2]
```diff
@@ -68,6 +68,9 @@ do
   fi
 done
 
+ENCODED_mongo_root_user=$mongo_root_user
```
Comment (mohanarpit) [inline deploy/aws/boot.sh:71]:
```suggestion
encoded_mongo_root_user=$mongo_root_user
```

Please follow naming conventions. Why have all-caps in the variable name here?

File: deploy/aws/boot.sh [hunk 3]
```diff
@@ -68,6 +68,9 @@ do
...
+ENCODED_mongo_root_password=$mongo_root_password
```
Comment (mohanarpit) [inline deploy/aws/boot.sh:72]:
```suggestion
encoded_mongo_root_password=$mongo_root_password
```

File: deploy/aws/boot.sh (hunk 2 above)
Comment (manish535) [reply deploy/aws/boot.sh]: just to highlight the `encoded` credential.

Comment (manish535) [reply deploy/aws/boot.sh]: Converted to lower_case

**Comment by mohanarpit** [comment]
Closing this as this has been addressed #310

**Pull request closed without merging**